src/game-logic/reducers/showNotification.js

```diff
@@ -0,0 +1,35 @@
+// TODO: Change showNotification to accept a configuration object instead of so
+// many formal parameters.
+/**
+ * @param {farmhand.state} state
+ * @param {string} message
+ * @param {string} [severity] Corresponds to the `severity` prop here:
+ * https://material-ui.com/api/alert/
+ * @returns {farmhand.state}
+ * @see https://material-ui.com/api/alert/
+ */
+export const showNotification = (
```

Comment:
tests 🏴

Reply:
It turns out that this actually had tests! I moved them into their own file in https://github.com/jeremyckahn/farmhand/commit/c337899e6c0ae332648d56a457bcc2c734fe3173. Thanks for calling this out!